### ISSUE TITLE
AE2Things mod suggestion

### ISSUE THREAD

**Issue by DmitryProskurin** [TheStaticVoid/StaTech-Industry#512]
I want to suggest this mod because of Refined storage like disks.
[My port to 1.19.2.](https://github.com/DmitryProskurin/AE2Things/tree/1.19.2-port)

**Comment by TheStaticVoid** [comment]
is there a cf link?

**Comment by DmitryProskurin** [comment]
@TheStaticVoid, no, i don't have account on Curse Forge. You can publish it by yourself, i don't have any against it.

**Comment by TheStaticVoid** [comment]
Would prefer for you to either post it yourself on CF or make a PR to the original mod and have them maintain the backport on their page.

**Comment by DmitryProskurin** [comment]
@TheStaticVoid, ok, i'll see what i can do.

**Comment by DmitryProskurin** [comment]
@TheStaticVoid [CurseForge version](https://www.curseforge.com/minecraft/mc-mods/ae2-things-fabric-1-19-2)

**Comment by TheStaticVoid** [comment]
Added in 1.1.14
